### ISSUE TITLE
Execution Id is missing in the UnitTestType

### ISSUE THREAD

**Issue by liuqidake** [ShuiRuTian/trx-reporter-for-playwright#10]
Azure devops 1Es test populates test result and owner information based on the execution id in a way that they will look for <Execution id=.... /> under <UnitTest>. Based on this id, it will search the UnitTestResult to find the result of the test case, after which the owner, className and result will be eventually grouped together. But currently addTestResult method under TestRunBuilder does not include the execution id when initializing UniTestType, causing the devops can't show any owner information. 

This is the code that I am referring:

const unitTest: UnitTestType = new UnitTestType({
      $id: testResult.$testId,
      $name: [className, testResult.$testName].join(NAME_SPLITTER),
      $priority: priority,
      $storage: fileLocation,
      Owners: owner ? {
        Owner: [{ $name: owner }],
      } : undefined,
      WorkItemIDs: workItemIds
        ? { WorkItem: workItemIds.map((item) => ({ $id: item })) }
        : undefined,
      TestMethod: {
        $className: className,
        $codeBase: fileLocation,
        $name: testResult.$testName,
      },
    });

**Comment by ShuiRuTian** [comment]
Sorry for the late response, I remember it's recommended to use Junit plugin for this situation now by official team.

Please correct me if I am wrong! Would be happy to see it and fix this issue.

**Comment by liuqidake** [comment]
@ShuiRuTian Do you mean the junit test reporter from playwright? That seems like not really working well with the owner information on Azure cloud test. Do you mind me sending a pull request here? It is just oneline code change by adding the execution id to the unittest tag

**Comment by ShuiRuTian** [comment]
@liuqidake 

>Do you mean the junit test reporter from playwright?
Yeah, I think so. But feels like I remember it wrongly, since you must have investigate it a bit.

> Do you mind me sending a pull request here?
Cool, why not?
